backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/I_C_Order.java

```diff
@@ -1019,6 +1020,33 @@ public interface I_C_Order
 	ModelColumn<I_C_Order, Object> COLUMN_CreateNewFromProposal = new ModelColumn<>(I_C_Order.class, "CreateNewFromProposal", null);
 	String COLUMNNAME_CreateNewFromProposal = "CreateNewFromProposal";
 
+	/**
+	 * Set Year.
+	 * Calendar Year
+	 *
+	 * <br>Type: Search
+	 * <br>Mandatory: false
+	 * <br>Virtual Column: false
+	 */
+	void setC_Year_ID (int C_Year_ID);
```

Comment:
as discussed, consider 
* removing it
* or moving it to customer branch (and rename it)
* or rename it to something like Contract_Year_ID

Reply:
removed from code. Since it was introduced outside of this issue, and because other issues are blocked waiting for this code, merging and we can follow-up on this next week.